### DISCUSSION TITLE
split border color/size change

### DISCUSSION THREAD

**Discussion by Robinx0** [ghostty-org/ghostty#3878] [Q&A] [answered]
hi, is there any config for the split border size/color change. if used darker background like in the image, the border isn't visible. 
maybe like OS-split-border-fill and border-size?

![Screenshot 2024-12-29 at 5 26 01 PM](https://github.com/user-attachments/assets/f8ee50e3-d84d-4ec4-b36b-cd73ab3ce94f)

**Comment by 00-kat** [comment, marked as answer]
Dupe of #3301.
